ARCollectionViewMasonryLayout.podspec

```diff
@@ -1,16 +1,17 @@
 Pod::Spec.new do |s|
-  s.name             = "ARCollectionViewMasonryLayout"
-  s.version          = "2.0.0"
-  s.summary          = "ARCollectionViewMasonryLayout is a UICollectionViewLayout subclass for creating flow-like layouts with dynamic widths or heights."
-  s.homepage         = "https://github.com/AshFurrow/ARCollectionViewMasonryLayout"
-  s.screenshots      = "https://raw.githubusercontent.com/AshFurrow/ARCollectionViewMasonryLayout/master/Screenshots/ARCollectionViewMasonryLayout.png"
-  s.license          = 'MIT'
-  s.author           = { "Orta Therox" => "orta.therox@gmail.com" }
-  s.social_media_url = "http://twitter.com/orta"
-  s.platform         = :ios
-  s.platform         = :ios, '7.0'
-  s.source           = { :git => "https://github.com/AshFurrow/ARCollectionViewMasonryLayout.git", :tag => "#{s.version}" }
-  s.source_files     = 'ARCollectionViewMasonryLayout.{h,m}'
-  s.framework        = 'UIKit'
-  s.requires_arc     = true
+  s.name                = "ARCollectionViewMasonryLayout"
+  s.version             = "2.0.0"
```

Comment:
It sure is API-wise (see the [changes to the public header](https://github.com/ashfurrow/ARCollectionViewMasonryLayout/pull/23/files#diff-17e892cfe7f27304974edd320ccddc46)), but the behaviour does change of course, so not sure how we should be labelling that under SemVer.